modules/core/client/config/core.client.config.google.analytics.js

```diff
@@ -0,0 +1,16 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('core')
+    .config(tracking)
+    .run(function(Analytics) {});
+
+  function tracking(AnalyticsProvider, CORE_CONSTANTS) {
+    AnalyticsProvider.setAccount(CORE_CONSTANTS.analytics);
+    AnalyticsProvider.trackPages(true);
+    AnalyticsProvider.trackUrlParams(true);
```

Comment:
// Track all URL query params (default is false).
    AnalyticsProvider.trackUrlParams(false);

dont think we need this set or set to false.